docs/extending-locust.rst

```diff
@@ -153,6 +153,14 @@ authentication to the app should be granted.
 A full example can be seen `in the auth example <https://github.com/locustio/locust/tree/master/examples/web_ui_auth.py>`_.
 
 
+Exporting Stats to InfluxDB
+===========================
+
+You may wish to have more contorl over how the stats are aggregated and displayed. With the `--exporter` flag, Locust will
+export the stats to InfluxDB. Simply set the environment variables `INFLUX_URL`, `INFLUX_TOKEN`, and `INFLUX_ORG` for your
```

Comment:
I'd prefer using the standard INFLUXDB_*-vars. 

Couldnt really find them properly documented but they are INFLUXDB_URL, INFLUXDB_TOKEN etc